### DISCUSSION TITLE
Developer blog

### DISCUSSION THREAD

**Discussion by rbotzer** [aerospike/spring-data-aerospike#519] [Team Posts]
So I know you folks have a day job, but please consider writing a small post about using Spring Data with Aerospike. We launched a publication on Medium for short form articles for developers.

https://medium.com/aerospike-developer-blog

We would be happy to add anything you have to the blog. Thanks!

**Comment by Aloren** [comment]
Great idea! We will prepare a post :) Thank you

**Comment by Aloren** [comment]
Hi @rbotzer ! 
I've started preparing blogpost with the demo project: https://github.com/aerospike-community/spring-data-aerospike-demo I will let you know when it is ready. Also if you have any specific topics that you think should be covered, please let me know. Thank you.

**Comment by rbotzer** [comment]
That's great. It would be useful to hit on some of the questions regarding Spring Data and Aerospike in Stackoverflow, as well as the basics.
Thank you.

**Comment by rbotzer** [comment]
I've moved the documentation for Spring Data from our docs.aerospike.com site to the wiki of the [`spring-data-aerospike`](https://github.com/aerospike-community/spring-data-aerospike/wiki) repo.

If you're not too far along with the blog post, I wanted to suggest that it should be posted to https://dev.to/ rather than Medium, because of the paywall issue on Medium. But really whatever works for you is great. Send me the URL and our developer advocate (Natalie Pistunovich [@pisush](https://github.com/pisush)) will write about it in her bi-monthly writeup.
